Create ui-c8y-1022-9-4-fixed-issue-with-wrong-device-source-for-alarms-and-events-fix.md (#3576)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1022-9-4-fixed-issue-with-wrong-device-source-for-alarms-and-events-fix.md

---
date: ""

> title: Fixed issue with wrong device source for alarms and events when using type dashboards.

Reviewer: ```suggestion
title: Alarms and events now linked to the correct device source
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63946
version: 1022.9.4
---

> In some cases, alarms and events were incorrectly associated with the wrong device for device type dashboards, leading to confusion and difficulties in troubleshooting. This issue has now been resolved. Alarms and events will be correctly linked to the originating device, ensuring accurate tracking and analysis. This change improves the reliability and usability of the alarm and event management system for all users.

Reviewer: ```suggestion
In some cases, alarms and events were incorrectly associated with the wrong device when using device type dashboards. This issue has been resolved. Alarms and events are now correctly linked to the originating device, ensuring accurate tracking and analysis. 
```

---
date: ""

> title: Fixed issue with wrong device source for alarms and events + fix

Reviewer 2: ```suggestion
title: Fixed issue with wrong device source for alarms and events when using type dashboards.
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63946
version: 1022.9.4
---

> Co-authored-by: Enio Sultan <92023325+eniosultan@users.noreply.github.com>
> Co-authored-by: Jakub Drewniak <103751383+jdre-c8y@users.noreply.github.com>

Reviewer 2: ```suggestion
In some cases, alarms and events were incorrectly associated with the wrong device for device type dashboards, leading to confusion and difficulties in troubleshooting. This issue has now been resolved. Alarms and events will be correctly linked to the originating device, ensuring accurate tracking and analysis. This change improves the reliability and usability of the alarm and event management system for all users.
```
